scripts/testnet/variables.sh

```diff
@@ -60,14 +60,16 @@ export GENESIS_STAKE_TYPE="direct" #'delegated' or 'direct' as in direct stake
 #if set to 1, each observer will turn off the antiflooding capability, allowing spam in our network
 export OBSERVERS_ANTIFLOOD_DISABLE=0
 
-# If set to true, this will deploy nodes in a sovereign shard.
+# If set to 1, this will deploy nodes in a sovereign shard.
 # All variables from metashard structure(validators, observers, consensus) should be set to zero and SHARDCOUNT to 1
 # For now, make sure that you checkout feat/sovereign branch from mx-chain-deploy repo when using these scripts
-export SOVEREIGN_DEPLOY=true
+export SOVEREIGN_DEPLOY=1
+
+export ROUND_DURATION=4000
```

Comment:
I don't think this is ok.
We are overwriting the RoundDuration from `nodesSetup.json` with this.
Is this really needed/expected?
cc @bogdan-rosianu @raduchis

Reply:
yes, we can revert. we needed a shorter round on the demo for a more responsive feedback

Reply:
yes, can be reverted. As @bogdan-rosianu mentioned it was updated for the xDay demo